src/src/CustomBuild/define_plugin_sets.h

```diff
@@ -391,6 +391,14 @@ To create/register a plugin, you have to :
     #ifndef FEATURE_RULES_EASY_COLOR_CODE
         #define FEATURE_RULES_EASY_COLOR_CODE 1
     #endif
+
+    #ifndef USES_P095
+        #define USES_P095   // ILI9341 , added by TEB
+    #endif
+
+    #ifndef USES_P110
+        #define USES_P110   // VL53L0X Time of Flight sensor, added by TEB
+    #endif

```

Comment:
These changes should have better been kept local, as these plugins won't fit in the ESP8266 builds

Reply:
Please disregard the changes to define_plugin_sets.